Wireframe/index.html

```diff
@@ -7,7 +7,32 @@
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-</body>
-</html
\ No newline at end of file
+    <header>
+      <h1>GitHub For Software Developers</h1>
+      <p>Understanding Git and its importance in the software development process.</p>
+    </header>
+    <main>
+      <section class=""articles">
```

Comment:
1. What is this section doing in your code? Can you talk through your reasoning?

2) It looks like there is a syntax error - can you spot it?

Reply:
1. This part is me  setting up the basic structure of the webpage and defining different parts of the document using semantic HTML elements. I’m using <main> to wrap the main content of my webpage, I am to using a <section> because it allows me to break my page into meaningful blocks.
2. yes the error here is the double quotations 
      <section class=""articles">, the correct one will be 
      <section class="articles">

Reply:
1. This part is me setting up the basic structure of the webpage and defining different parts of the document using semantic HTML elements. I’m using -main- to wrap the main content of my webpage, I am to using a -section-
because it allows me to break my page into meaningful blocks.
2. yes the error here is the double quotations i have ""...."